pyre/extra_tests/parity_tests/list_pop_fold.py

```
@@ -116,6 +116,31 @@ def pop_object(n):
 
 
 # ── despecialisation mid-loop: an Integer list that becomes an Object one ──
+class Box(object):
```

Ruff (0.16.2)

[warning] Class `Box` inherits from `object`

Remove `object` inheritance

(UP004)